lib/periodicChecks.js

```diff
@@ -174,7 +175,38 @@ const ensureAllIssuesHaveProjects = async (context) => {
   });
 };
 
+/**
+ * 1. Check if the new issue have a project assigned.
+ * 2. If not then it will ping Core Maintainers
+ * @param {import('probot').Context} context
+ */
+const ensureNewIssuesHaveProjects = async(context) =>{
+  await sleep(20000);
```

Comment:
Why this sleep?

Reply:
This sleep is necessary here, Because this is providing time to user to add project, Otherwise if this is not used then as soon as Issue gets created Oppiabot will comment to add project and will not provide user time to add project before.

Reply:
Hmm, okay makes sense, this is 20 second right?

Reply:
Probably add a comment in the code about this.

Reply:
Sure Sir, Adding them right now.

Reply:
Is there any other change required Sir, Or Should I perform these changes and perform PR?

Reply:
> Hmm, okay makes sense, this is 20 second right?

Yes sir.